pkg/apis/stackconfigpolicy/v1alpha1/stackconfigpolicy_types.go

```diff
@@ -0,0 +1,210 @@
+	// SnapshotLifecyclePolicies holds the Snapshot Lifecycle Policies settings (/_slm/policy)
+	// +kubebuilder:pruning:PreserveUnknownFields
+	SnapshotLifecyclePolicies *commonv1.Config `json:"snapshotLifecyclePolicies,omitempty"`
+	// SecurityRoleMappings holds the Role Mappings settings (/_security/role_mapping)
+	// +kubebuilder:pruning:PreserveUnknownFields
+	// SecurityRoleMappings *commonv1.Config `json:"securityRoleMapping,omitempty"`
+	// AutoscalingPolicies holds the Autoscaling Policies settings (/_autoscaling/policy)
+	// +kubebuilder:pruning:PreserveUnknownFields
+	// AutoscalingPolicies *commonv1.Config `json:"autoscalingPolicies,omitempty"`
+	// IndexLifecyclePolicies holds the Index Lifecycle policies settings (/_ilm/policy)
+	// +kubebuilder:pruning:PreserveUnknownFields
+	// IndexLifecyclePolicies *commonv1.Config `json:"indexLifecyclePolicies,omitempty"`
+	// IngestPipelines holds the Ingest Pipelines settings (/_ingest/pipeline)
+	// +kubebuilder:pruning:PreserveUnknownFields
+	// IngestPipelines *commonv1.Config `json:"ingestPipelines,omitempty"`
+	// IndexTemplates holds the Index and Component Templates settings
+	// IndexTemplates *IndexTemplates `json:"indexTemplates,omitempty"`
+}
+
+/*type IndexTemplates struct {
+	// ComponentTemplates holds the Component Templates settings (/_component_template)
+	// +kubebuilder:pruning:PreserveUnknownFields
+	ComponentTemplates *commonv1.Config `json:"componentTemplates,omitempty"`
+	// ComposableIndexTemplates holds the Index Templates settings (/_index_template)
+	// +kubebuilder:pruning:PreserveUnknownFields
+	ComposableIndexTemplates *commonv1.Config `json:"composableIndexTemplates,omitempty"`
+}*/
+
+type StackConfigPolicyStatus struct {
+	// ResourcesStatuses holds the status for each resource to be configured.
+	ResourcesStatuses map[string]ResourcePolicyStatus `json:"resourcesStatuses"`
+	// Resources is the number of resources to be configured.
+	Resources int `json:"resources,omitempty"`
+	// Ready is the number of resources successfully configured.
+	Ready int `json:"ready,omitempty"`
+	// Errors is the number of resources which have an incorrect configuration
+	Error int `json:"errors,omitempty"`
+	// ReadyCount is a Human representation of the number of resources successfully configured.
+	ReadyCount string `json:"readyCount,omitempty"`
+	// Phase is the phase of the StackConfigPolicy.
+	Phase PolicyPhase `json:"phase,omitempty"`
+	// ObservedGeneration is the most recent generation observed for this StackConfigPolicy.
+	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
+}
+
+type PolicyPhase string
+
+const (
+	UnknownPhase         PolicyPhase = "Unknown"
+	ReadyPhase           PolicyPhase = "Ready"
+	ApplyingChangesPhase PolicyPhase = "ApplyingChanges"
+	InvalidPhase         PolicyPhase = "Invalid"
+	ErrorPhase           PolicyPhase = "Error"
+	ConflictPhase        PolicyPhase = "Conflict"
+)
+
+// phaseOrder
```

Comment:
Was this comment intended to add more information, or does it need to be removed?  I'd be in favor of more information on how this is used.

Reply:
It's an oversight, I will add information.